boards/arm/particle_argon_nrf52/particle_argon_nrf52.dts

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2018 Matthias Boesl
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+
+/ {
+	model = "particle argon nrf52";
+	compatible = "particle,particle_argon", "nordic,nrf52840-qiaa",
+		     "nordic,nrf52840";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		user_led: led_0 {
+			gpios = <&gpio1 12 0>;
+			label = "User LED";
+		};
+		status_green: led_1 {
```

Comment:
Please re-order these for led1=red, led2=green, led3=blue, following pin order and common practice of RGB.  (The current one matches the original xenon order in #11587, but I've asked for that to change too).